StandaloneMmPkg/Drivers/StandaloneMmIplPei/StandaloneMmIplPei.inf

```diff
@@ -0,0 +1,80 @@
+## @file
+#  This module provide a Standalone MM compliant implementation of MM IPL PEIM.
+#
+#  Copyright (c) 2024, Intel Corporation. All rights reserved.<BR>
+#
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = StandaloneMmIplPei
+  FILE_GUID                      = 578A0D17-2DC0-4C7D-A121-D8D771923BB0
+  MODULE_TYPE                    = PEIM
+  VERSION_STRING                 = 1.0
+  PI_SPECIFICATION_VERSION       = 0x0001000A
+  ENTRY_POINT                    = StandaloneMmIplPeiEntry
+
+#
+# The following information is for reference only and not required by the build tools.
+#
+#  VALID_ARCHITECTURES           = IA32 X64
+#
+
+[Sources]
+  StandaloneMmIplPei.c
+  StandaloneMmIplPei.h
+  MmFoundationHob.c
+
+[Packages]
+  MdePkg/MdePkg.dec
+  MdeModulePkg/MdeModulePkg.dec
+  StandaloneMmPkg/StandaloneMmPkg.dec
+  UefiCpuPkg/UefiCpuPkg.dec
+
+[LibraryClasses]
+  PeimEntryPoint
+  PeiServicesTablePointerLib
+  PeiServicesLib
+  DebugLib
+  HobLib
+  MemoryAllocationLib
+  MmUnblockMemoryLib
+  BaseMemoryLib
+  PeCoffLib
+  CacheMaintenanceLib
+  MmPlatformHobProducerLib
+
+[Guids]
+  gEfiSmmSmramMemoryGuid
+  gEdkiiCommunicationBufferGuid
+  gEdkiiEventMmDispatchGuid
+  gSmmBaseHobGuid
+  gMpInformation2HobGuid
+  gEdkiiSmmProfileDataGuid
+  gEfiAcpiVariableGuid
+  gMmUnblockRegionHobGuid
+  gEdkiiAcpiS3EnableHobGuid
+  gEdkiiSmmCpuSyncConfigHobGuid
+
+[Ppis]
+  gPeiSmmControlPpiGuid
+  gEfiPeiMmCommunicationPpiGuid
+  gEfiEndOfPeiSignalPpiGuid
+
+[Protocols]
+  gEfiMmEndOfPeiProtocol
+
+[Pcd]
+  gEfiMdeModulePkgTokenSpaceGuid.PcdFixedCommBufferPages
+  gUefiCpuPkgTokenSpaceGuid.PcdCpuSmmRestrictedMemoryAccess        ## CONSUMES

```

Comment:
Move to [Pcd.X64] for PcdCpuSmmRestrictedMemoryAccess, else will cause build failure        

[Pcd.X64]
  gUefiCpuPkgTokenSpaceGuid.PcdCpuSmmRestrictedMemoryAccess        ## CONSUMES

Reply:
Not meet this, which build will fail ?